server/service/src/pricing/mod.rs

```diff
@@ -0,0 +1,54 @@
+use crate::service_provider::ServiceContext;
+use discount::{get_discount_for_item_and_name_link_id, ItemDiscountLookup};
+use repository::{
+    EqualFilter, MasterList, MasterListFilter, MasterListRepository, NameLinkRowRepository,
+    RepositoryError, StorageConnection,
+};
+
+pub mod discount;
+
+pub trait PricingServiceTrait: Sync + Send {
+    fn get_discount_for_item_and_name_link_id(
+        &self,
+        ctx: &ServiceContext,
+        input: ItemDiscountLookup,
+    ) -> Result<f64, RepositoryError> {
+        get_discount_for_item_and_name_link_id(ctx, input)
+    }
+}
+
+pub struct PricingService {}
+impl PricingServiceTrait for PricingService {}
+
+fn get_default_price_list_for_store(
```

Comment:
Will remove or move this logic before the final PR...
Just kept it here for reference for now...